R/shoji-folder.R

```diff
@@ -120,8 +120,30 @@ setMethod("delete", "ShojiFolder", function(x, ...) {
     if (is.null(parentFolderURL(x))) {
         halt("Cannot delete root folder")
     }
+
+    # count the variable/folder objects, and warn the user that they will be
+    # summarily deleted as well. Projects must be empty to be deleted (which is
+    # enforced on the server, so we only need to check VariableFolders) send as
+    # a message before the prompt for test-ability, and so the prompt isn't lost
+    # at then end of a long line.
+    if (inherits(x, "VariableFolder")) {
+        obj_names <- names(x)
+        num_vars <- length(x)
+        obj_word <- if(num_vars > 1) "objects" else "object"
```

Comment:
```suggestion
        obj_word <- ifelse(num_vars > 1, "objects", "object")
```

inline `if (something) x else y` hurts my eyes